docs/api/ref/Metadata.md

```diff
@@ -72,12 +74,12 @@ Optionally, extra information from the app directory can be returned, to aid in
 In situations where a desktop agent connects to multiple app directories or multiple versions of the same app exists in a single app directory, it may be necessary to specify `appId` or `version` to target applications that share the same name.
```

Comment:
```suggestion
In situations where a Desktop Agent connects to multiple app directories or multiple versions of the same app exists in a single app directory, it may be necessary to specify `appId` or `version` to target applications that share the same name.
```